SSW.Rules.AzFuncs/Functions/Bookmarks/RemoveBookmarkFunction.cs

```diff
@@ -22,44 +20,32 @@ public class RemoveBookmarkFunction(
     [Function("RemoveBookmarkFunction")]
     public async Task<HttpResponseData> Run(
         [HttpTrigger(AuthorizationLevel.Anonymous, "post", Route = null)]
-        HttpRequestData req,
+        HttpRequestData request,
         FunctionContext executionContext)
     {
-        ApiAuthorizationResult authorizationResult =
-            await apiAuthorization.AuthorizeAsync(Converters.ConvertToIHeaderDictionary(req.Headers));
-
+        var authorizationResult = await apiAuthorization.AuthorizeAsync(Converters.ConvertToIHeaderDictionary(request.Headers));
         if (authorizationResult.Failed)
         {
             _logger.LogWarning(authorizationResult.FailureReason);
-            return req.CreateJsonErrorResponse(HttpStatusCode.Unauthorized);
+            return request.CreateJsonErrorResponse(HttpStatusCode.Unauthorized);
         }
-
         _logger.LogWarning($"HTTP trigger function {nameof(RemoveBookmarkFunction)} request is authorized.");
 
-        Bookmark data;
-
-        string requestBody = await new StreamReader(req.Body).ReadToEndAsync();
-        data = JsonConvert.DeserializeObject<Bookmark>(requestBody);
-
-        bool isNull = string.IsNullOrEmpty(data?.RuleGuid) || string.IsNullOrEmpty(data?.UserId);
-        if (data == null || isNull)
+        var requestBody = await new StreamReader(request.Body).ReadToEndAsync();
+        var data = JsonConvert.DeserializeObject<Bookmark>(requestBody);
+        if (data is null || string.IsNullOrEmpty(data?.RuleGuid) || string.IsNullOrEmpty(data?.UserId))
         {
-            return req.CreateJsonResponse(new
-            {
-                error = true,
-                message = "Request body is empty",
-            }, HttpStatusCode.BadRequest);
+            return request.CreateJsonErrorResponse(HttpStatusCode.BadRequest, "Request body is empty");
         }
 
-        var results =
-            await dbContext.Bookmarks.Query(q => q.Where(w => w.RuleGuid == data.RuleGuid && w.UserId == data.UserId));
-
-        var model = results.FirstOrDefault();
-
-        if (model == null)
+        var results = await dbContext
+            .Bookmarks
+            .Query(q => q.Where(w => w.RuleGuid == data.RuleGuid && w.UserId == data.UserId));
+        var existingBookmark = results.FirstOrDefault();
+        if (existingBookmark is null)
         {
-            _logger.LogInformation("No bookmark exists for User {0} and Rule {1}", data.UserId, data.RuleGuid);
-            return req.CreateJsonResponse(new
+            _logger.LogInformation($"No bookmark exists for User {data.UserId} and Rule {data.RuleGuid}");
```

Comment:
1. Read
https://learn.microsoft.com/en-us/dotnet/core/extensions/logging-library-authors#avoid-string-interpolation-in-logging

Reply:
✅ Done, changed to log message template